server/config.go

```diff
@@ -424,6 +427,7 @@ const (
 	defaultMaxPendingPeerCount  = 16
 	defaultMaxMergeRegionSize   = 20
 	defaultSplitMergeInterval   = 1 * time.Hour
+	defaultPatrolRegionInterval = 100 * time.Millisecond
```

Comment:
Can we make 10ms?

Reply:
I think we can turn lower when need speed up. in `10ms`, it consumes more CPU. I do pprof and found the `proto.Clone()` is also a problem.In test cluster `100ms` change to `10 ms`, the CPU growth about 60%.